### ISSUE TITLE
does the play button not exist?

### ISSUE THREAD

**Issue by rossman22590** [enricoros/nextjs-chatgpt-app#40]
i do not see the play button anymore on code blocks, what can id o to get it back?

**Comment by rossman22590** [comment]
if i can have it back somehow i will be happy!

**Comment by rossman22590** [comment]
i went back a  commit and found this , are you removing forever?

**Comment by enricoros** [comment]
Correct, I decided to remove the run button because I didn't see people or happy people using it, just complaints..also, that library was doubling the size of the application, from 198 to 400k of JS.

In the meantime, this is the last deployment that has it:
https://nextjs-chatgpt-hre929hjj-enricoros.vercel.app/

Let me know your use case and how satisfied you are with the run button, and I can revert the commit and add the button back.
